theme/styles/snippets/markdown.scss

```diff
@@ -90,13 +90,13 @@
     }
 
     td:first-of-type {
-      font-family: $itc-garamond;
+      font-family: $serif;
       @extend %serif-lg;
     }
 
     td:nth-of-type(2) {
-      font-family: $abc-diatype;
-      @extend %sans-sm;
+      font-family: $sans-serif;
+      @extend %sans-light-sm;
```

Comment:
shouldn't `%sans-light-sm` already have the font-family applied. It feels like the prefix `sans-` implies that the styling already includes the ``sans-serif` font-family

Reply:
Yes! fixed this :)